test/widgets.js

```diff
@@ -263,14 +258,6 @@ describe('Widgets', function() {
         assert(result.includes('<li>widget4 - time: 15:39:12</li>'));
       });
 
-      it('should not render the placeholders when widget\'s fields are defined', function() {
-        assert(result.includes('<li>widget5 - string: Some string</li>'));
-        assert(result.includes('<li>widget5 - integer: 2</li>'));
-        assert(result.includes('<li>widget5 - float: 2.2</li>'));
-        assert(result.includes('<li>widget5 - date: 2022-09-21</li>'));
-        assert(result.includes('<li>widget5 - time: 15:39:12</li>'));
-      });
-
```

Comment:
this test doesn't pass anymore? due to the change in sanitize method?

Reply:
Correct. We don't care about user-entered values at all yet if `aposPlaceholder` is still set and the field has a `placeholder`.